kornia-py/src/tracking/lucas_kanade.rs

```diff
@@ -0,0 +1,37 @@
+use glam::{Vec2, Mat2};
+use kornia::filters::sobel;
+use kornia::tensor::Tensor;
+use tch::{Device, Kind};
+
+/// Implements Lucas-Kanade Optical Flow using glam-rs
+pub fn lucas_kanade(
+    prev_img: &Tensor,
+    next_img: &Tensor,
+    points: &[(f32, f32)],
+    win_size: usize,
+) -> Vec<(f32, f32)> {
+    let mut flow = Vec::new();
+
+    for &(x, y) in points {
+        let mut sum_dx = Vec2::ZERO;
+        let mut sum_dy = Vec2::ZERO;
+
+        for i in 0..win_size {
+            for j in 0..win_size {
+                // Calculate gradients using Kornia's Sobel filter
+                let dx = sobel(prev_img, "x", true);
```

Comment:
I don't think you never tried to compile this code as this method doesn't exist or defined here